uw-frame-components/js/frame-config.js

```diff
@@ -30,7 +46,11 @@ define(['angular'], function(angular) {
             "subtitle" : "beta",
             "ariaLabelTitle" : "My U W",
             "crestalt" : "UW Stevens Point Logo",
-            "group" : "UW System-Stevens Point"
+            "group" : "UW System-Stevens Point",
+            "footerLinks":[{ "url" : "https://kb.wisconsin.edu/myuwsystem/",
```

Comment:
Status quo is to have a help link

![status quo system portal help](https://cloud.githubusercontent.com/assets/952283/16585677/b64620cc-4287-11e6-8384-33f6a6f2315c.png)

that goes to [in-portal help](https://my.wisconsin.edu/portal/p/myuw-system-help).

![myuw system help](https://cloud.githubusercontent.com/assets/952283/16585733/d5f9ca18-4287-11e6-88ab-d29d6b5e189a.png)

This changeset would switch it up so that Help goes directly to the KB. Cutting out the interstitial to improve the user help experience?